utils/cosmosdb.ts

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
We won't need cosmos in this project, we can remove this module

Reply:
Fix with 066daa4